Duckling/Numeral/KA/Corpus.hs

```diff
@@ -35,9 +37,378 @@ allExamples = concat
   , examples (NumeralValue 2)
              [ "2"
              , "ორი"
+             , "წყვილი"
+             , "წყვილები"
              ]
   , examples (NumeralValue 3)
              [ "3"
              , "სამი"
+             , "ცოტა"
```

Comment:
I don't know if ცოტა, რამდენიმე etc is commonly used as three in Georgian. Is that true?

From what I understand these mean "a few" etc -- cc @patapizza how do we deal with that in other languages? -- "a little bit" "a few" etc

Reply:
@stopachka Ha got it, in English we also do "a few" = 3 :)